{% extends "layout.html" %}

{% block title %}Attributions{% endblock %}
{% block bnrurl %}https://static.igem.wiki/teams/4509/wiki/banners/attributions.jpeg{% endblock %}
{% block page_content %}
<style>
  .title {
    color: #7e5366;
    font-family: 'LucidaUnicode';
  }

  h3 {
    text-align: justify;
    text-justify: inter-word;
    color: #7e5366;
    font-family: 'LucidaUnicode';
    margin-bottom: 5%;
  }
</style>

<div class="bd-callout bd-callout-info">
  <h2>Overview</h2>
  <p>iGEM is a true team sport that extends beyond students tackling laboratory
    apparatus, equations, and computation, and connecting web mails to the
    pronounced community of researchers, professors, colleagues, and neighbours. The
    iGEM REC-CHENNAI team is very grateful for the support and advice that were
    given to us throughout the 2022 spell. We would also like to thank everyone inside
    and outside the testing ground who assisted us to deliver Project Curlim to the 2022

    Grand Jamboree!!!</p>
</div>
<div class="container">
  <div class="container">
    <hr>
  </div>
</div>
<div>
  <h3>Project Curlim Investigators</h3>
  <p>
  <h6 class="lead" style="font-weight:700">DR. GAYATHRI VIJAYAKUMAR:</h6> Dr Gayathri is the Primary PI of iGEM REC-CHENNAI
  2022. She has aided us since the ideation stage, finding solutions to problems that have arisen during our
  project scaling work. She has actively assisted us with our orders and ensured that we received all the
  necessary components. <br><br>

  <h6 class="lead" style="font-weight:700">DR. JAYASREE HAREENDRAN:</h6>
  Dr Jayasree is the Secondary PI of team iGEM REC Chennai 2022. She
  guided us in the brainstorming process and urged us to think about the underlying plot and the bigger
  picture of our idea. She also encouraged us to prepare a list of specific tasks to be completed and to think
  about what we could reasonably do over the period.
  </p>

  <br>

  <h3>Project Curlim Instructors</h3>
  <p>
  <h6 class="lead" style="font-weight:700">DR. SATHYA K: </h6>
  Dr. Sathya furnished us with valuable suggestions on reactor design and supported us in
  making improvements during our project implementation. <br><br>

  <h6 class="lead" style="font-weight:700">PROF. SHREENIDHI K S: </h6>
  Professor Shreenidhi offered a broad view in genetic engineering, assisted us in
  establishing the theme for our project, and patiently reminded us of our commitments.
  </p>

  <br>

  <h3>The Team Behind Curlim!</h3>
  <p>
  <h6 class="lead" style="font-weight:700">MARIE JYOTSNA JAWAHAR (Team Lead):</h6>
  Project Administration, Led Project Conceptualization and
  Foundational Assessment, Part analysis and registry, iGEM Regulation Analysis, Wet Lab, Collaboration, Wiki content
  editor, Part Validation and Interacted with Experts
  & other iGEM teams. <br><br>

  <h6 class="lead" style="font-weight:700">VISHAL AMITH KANNAN (Team Lead):</h6>
  Project Administration, Led Wiki Studies alongside Graphic Design,
  Maintenance of Timeline, Wetlab Lead, Wiki design, Part validation Head, logo design and Project Investigation Team. <br><br>

  <h6 class="lead" style="font-weight:700">VIBHA MURALI:</h6>
  Lead of project development, led project modelling (iTASSER, PyMOL, alphafold), project reflection, Part Analyst, part
  validation team, in charge of part improvement.<br><br>

  <h6 class="lead" style="font-weight:700">RHEA HARRY WILLIAM:</h6>
  Lead of project development, led project reflection, project modelling(iTASSER, PyMOL, alphafold), Expert interaction,
  part validation team, in charge of Part Registry.
  <br><br>

  <h6 class="lead" style="font-weight:700">NETHRA L: </h6>
  Lead of human practice, collaborations, In-charge of wiki and science communication, wet lab,reactor team, led education and Outreach.
  <br><br>

  <h6 class="lead" style="font-weight:700">AVENDHIKA RAJA:</h6>
  Public Engagement, Wiki primary content writer and editor, part of wiki studies and co-lead iGEM Regulation Analysis, wet lab assistance, aided with human practices and collaborations, trip and housing.
  <br><br>

  <h6 class="lead" style="font-weight:700">A ANUKAVITHA:</h6>
  Construction of Parts - Vector Cloning, Part construction (Benchling), Helped with Part improvement,Sponsorship campaign , Wiki,content, logo design, and Human Practices Plan & Creative Head. <br><br>

  <h6 class="lead" style="font-weight:700">GRIDDALURU VINEETHA NAIDU: </h6>
  Responsible for Maintaining Minutes of Meetings, Part validation, Team Bio Brick Assembly, Lab Assistance. <br><br>

  <h6 class="lead" style="font-weight:700">PARINITHA K:</h6>
  Construction of Part - Vector Cloning, Wet Lab, Analysed and Drafted Safety
  Regulations and Experimental Design, Lab Maintenance, Content strategist, and Assisted with Entrepreneurship. <br><br>

  <h6 class="lead" style="font-weight:700">PONTIANA RITIKA C:</h6>
  Assistance in Data Curation, Background Research Analysis, Reactor Hardware Design, Part
  Validation, Wet Lab, Reactor simulation and construction, Responsible for Project Promotion & Visualization and Video Lead. <br><br>

  <h6 class="lead" style="font-weight:700">MOHAMMED ASLAM:</h6>
  Chemical Analysis, Assistance with Protocol, HP Assistance, Developing Industrial Models,
  Safety Regulation Analysis and Wet Lab Lead. <br><br>

  <h6 class="lead" style="font-weight:700">MOHAMMED JASSIM: </h6>
  Responsible for Wet lab Safety & Collecting Experimental Data, Wet Lab Experiments
  Peer-Reviewed Research Articles and Lab Maintenance. <br><br>

  <h6 class="lead" style="font-weight:700">ABHITHA K.V:</h6>
  Team Bio Brick Assembly, Assistance in Part Construction, Industrial Reactor Designing, Outreach
  Lead, Wetlab, Lab maintenance, Collaboration, Creative content writing and Managing Bills. <br><br>

  <h6 class="lead" style="font-weight:700">THUHIN KHANNA R:</h6>
  Responsible for Wiki Design and Development, Constructed Presentation and
  Communication Materials, Curcoli Game Developer. <br><br>

  <h6 class="lead" style="font-weight:700">SURAJ N:</h6>
  Responsible for Wiki Coding and Designing, Graphic Designing and Video Support, Curcoli Game Developer. <br><br>

  <h6 class="lead" style="font-weight:700">VARUN RISHWANDH S:</h6>
  Curcoli Game Developer, Wiki study's contributor, Assisted with Wiki Development & Designing and Video Support.
  </p>

  <br>

  <h3>Project Mentors</h3>
  <p>
    <span style="font-weight: 700;">DR. DURAIMURUGAN N</span> from the Department of Computer Science and Engineering,
    REC was a major mentor in
    the Development of the Curlim Wiki Page and the Curcoli Game. <br><br>

    <span style="font-weight: 700;">POOJA S</span>, <span style="font-weight: 700;">HARIKUMAR PADMARAJA</span>, and
    <span style="font-weight: 700;">VARSHINY GOPINATH</span> are the institutions' former iGEM team
    members who extended their Helping hands to the 2022 team in Areas such as Briefing iGEM Regulations, and Major
    contributions to Project Conceptualization, Project Investigation, Providing E. coli Strains, and Conducting
    Brainstorming Sessions. <br><br>

    <span style="font-weight: 700;">PADMASHREE B</span> is a current Post Graduate student at the Department who mentors
    the team on Reactor literature studies.
  </p>
</div>

<br>

<div class="bd-callout bd-callout-info">
  <h2>MORE SPECIFICALLY, WE'D LIKE TO THANK!</h2>
  <hr>
  <h3 class="mb-2">General Support</h3>
  <p>
    Our Deepest Gratitude to <a href="https://www.rajalakshmi.org/">Rajalakshmi Engineering College's</a> Chairperson, Vice Chairperson
    Principal, and the Management. <br><br>
    We are thankful for the assistance and support provided by our advisors in managing the Team, the Project, and the
    Experiments. <br><br>
    We thank the <a href="https://www.rajalakshmi.org/dept-biotech-intro.php" target="_blank">Department of
      Biotechnology</a>, Rajalakshmi Engineering College for supporting and encouraging Project Curlim to participate in
    and represent at the
    Grand Jamboree 2022. A special thanks to the most respected HOD, DEAN, Academic Heads, and Faculty Members from the
    Department of Biotechnology, REC.<br><br>
    We thank the <a href="https://www.rajalakshmi.org/dept-cse-intro.php" target="_blank">Department of Computer
      Science</a>, Rajalakshmi Engineering College for supporting and encouraging Project Curlim with Wiki page. A
    special thanks to the most respected HOD. <br> <br>
  </p>

  <h3 class="mb-2">Strain Support</h3>
  <p>
    We extend our gratitude towards Sathyabhama University for supplying <i>E.coli</i> DH5 (Alpha) for the team's wet
    lab work. <br><br>
    We would like to thank IIT Madras for providing <i>E.coli</i> DH5 (Alpha) strain and <i>E.coli</i> K12 strain.
  </p>
  <h3 class="mb-2">Project Consultation & Advice</h3>
  <p>
    We would like to thank Dr. Venkat Nancharaiah of the WSCD, BARC Facilities in Kalpakkam for providing the Team REC
    Chennai 2022 with a unique outlook on Reactor Construction. We would also like to thank Dr Kiran Kumar Reddy for
    advice on optimum maintenance of biofilm.
    <br><br>
    Thanks to Dr. B. Anandan, Assistant Professor at Madras University's Tharamani Campus, for providing us with
    constructive feedback on the environmental and
    reactor conditions to consider, along with his advice on Enzyme Kinetics and helping us with the primer synthesis.
    <br><br>
    Our heartfelt special thanks to Ms. Madalynn Thompson, a Ph.D. student at Dalhousie University in Halifax, Nova
    Scotia, Canada, for her assistance with the Project Criteria Standpoint.
    <br><br>
    <a href="https://www.linkedin.com/in/dr-mullai-venthan-selvam-19863713a/"> Dr. S. Mullai Venthan Selvam</a>,
    Assistant Professor, Department of Mathematics, REC, was instrumental in the development of Mathematical Computation
    and Equations.
    <br><br>
    We would like to thank Mr. Vijay Bhoopalan for advice on plasmid expression
    <br><br>
    We would like to thank the faculty members from the Department of Biotechnology, REC for advising the team on a
    variety of matters.
    <br><br>
    Assistant Professor <a href="https://www.linkedin.com/in/saravanan-periasamy-8816381b">Dr. P. Saravanan</a>
    instructed the team on the variable circumstance requisite for the E. coli Biofilms.
    <br><br>
    Assistant Professor <a href="https://www.linkedin.com/in/arun-arumugaperumal-44806813a">Dr. Arun Arumugaperumal</a>
    instructed the team regarding molecular overexpression of a targeted gene.
    <br><br>
    <a href="https://www.linkedin.com/in/dr-anitha-thulasi-54170bb9">Dr. T. Anitha</a>, Associate Professor, and <a
      href="https://www.linkedin.com/in/anila-ashly-ab124612b">Dr. R. Anila</a>, Associate Professor, prescribed the
    team on Reactor design elements.
    <br><br>
    <a href="https://www.linkedin.com/in/saranya-arunkumar-59789386">Ms. S. R. Saranya ArunKumar</a>, Assistant
    Professor, recommended certain transformation techniques.
    <br><br>
    <a href="https://www.linkedin.com/in/selvakumar-ta-319a78171/">Dr. Selvakumar T.A.</a>, Associate Professor, advised the team on initial <i>E.Coli</i> conditions.
  </p>
  <h3 class="mb-2">Financial Assistance</h3>
  <p>
    We would like to thank the REC Management for their support with the Registration Fees for this competition. <br>
    We sincerely thank SPAHJ INDIA TRADING PVT LTD for offering funds for our project this year.
  </p>
  <h3 class="mb-2">Collaboration & Technique Aid</h3>
  <p>
    Thanks to Various iGEM Teams of 2022 who Collaborated with Team Curlim. <br><br>
    Thanks to Deepak Kumar, iGEM REC Alumni.
    <br><br>
    We also thank the UG Students from REC:
    Poornima N, Department of Biotechnology, REC, for Connecting with Madalynn Thompson from Dalhousie University,
    Halifax, Nova Scotia, Canada
    ; Krithickraj, Department of Biotechnology, REC - Wave of Ideas Hp: Photographer
    <br><br>
    Promotion Video Attribution: Voice over Artist - Leonard and Rohith
  </p>
  <h3 class="mb-2">Lab Assistance</h3>
  <p>
    Thanks to REC Management for providing us with the necessary supplies.
    <br><br>
    We are grateful to the Department of Biotechnology, REC for providing the experiment platform, supplies, and tools.
    <br><br>
    We would like to express our appreciation to Medox Biotech India Pvt. Ltd, Twist Bioscience, New England BioLabs
    Inc., Integrated DNA Technologies IDT, Karthik
    Chemical Industries, and Eppendorf India Pvt. Ltd for helping us with our laboratory equipment needs.
    <br><br>
    We acknowledge the Department Firm's Lab Technicians' aid in carrying out our experimental procedures and providing
    the laboratory equipment.
    <br><br>
    We would like to thank <a href="https://www.hetrogenebiotech.com/">HetroGene Biotech</a> for providing us with
    additional biosafety training.
  </p>
  <h3 class="mb-2">Human Practice Support</h3>
  <p>
    Wave of Ideas: Tamil Nadu Government, Greater Chennai Corporation, Commissioner of Police- Chennai, Deputy
    Commissioner of Police- Adyar, Chennai. <br>
    Location: Edward Elliot's Beach
    <br><br>
    We thank the following volunteers from Department of Biotechnology, REC who extended a helping hand in our beach human-practice event, Jayashree, Jebisha Jose, Priyanga, Ram kumar, Rooshini and Saikumar. We would also like to thank Jothimurugan, Kadhiravan, Krithik
    Ragunath, Mukesh Karthik, Naresh, Naveenkumar, Sai Sandhya Sree and Saranya for helping us during the event.
    <br><br>
    Human Practice in Educational Institutions: We would like to express our gratitude to the Management, Staff, and Members of the Educational Foundations of
    SBOA School and Junior College, Sacred Heart Matric. Hr, Sec. School and Bhavans Rajaji Vidyashram CBSE School for allowing us to interact with students for the following HP's The SynBio Connect, Enlighting yound minds and The DNAwesome Talk.
  </p>
</div>

<hr>

<div>
  <h2>SPONSORS</h2>
  <div class="container p-4" style="background-color: #ffffff;">
    <center>
      <img class="img p-2" style="width: 300px;" src="https://static.igem.wiki/teams/4509/wiki/sponsors/bench.png">
      <img class="img p-2" style="width: 300px;" src="https://static.igem.wiki/teams/4509/wiki/sponsors/math.jpg">
      <img class="img p-2" style="width: 300px;" src="https://static.igem.wiki/teams/4509/wiki/sponsors/sponsor.png">
      <img class="img p-2" style="width: 300px;"
        src="https://static.igem.wiki/teams/4509/wiki/sponsors/rajalakshmi.png">
      <img class="img p-2" style="width: 300px;" src="https://static.igem.wiki/teams/4509/wiki/sponsors/twist.png">
      <img class="img p-2" style="width: 300px;" src="https://static.igem.wiki/teams/4509/wiki/sponsors/medox.jpg">
      <img class="img p-2" style="width: 300px;" src="https://static.igem.wiki/teams/4509/wiki/sponsors/idt.png">
    </center>
  </div>
</div>

<br>
<hr>

<div>
  <h2>TEAM BUILDING AND PROJECT LAUNCH</h2>
  <p>
    We are a group of undergraduates from Rajalakshmi Engineering College who are currently enrolled in the 4-year
    B.Tech. Biotechnology program. Synthetic biology is
    one of the key areas that our institution offers alongside other subjects.
    <br><br>
    The foundation for the team was laid during Spring 2022, through a series of selection processes, interviews, and
    talks from former iGEM teams on the project
    proposal and competition requirements.
    <br><br>
    We began brainstorming in March of this year, generating conceptual plans for the project. The arrangement of
    precise tasks and scheduling began in the Summer of

    2022, accompanied by the Wet-lab experiment.
  </p>
</div>

<div id="references1">
  <h3>References <hr></h3>
  <p>We would like to thank all the scientist and authors for their contributions in the field of science that made this prooject possible.</p> <br>
  <p>
    1. Ali, H., & Khan, E. (2019). Trophic transfer, bioaccumulation, and biomagnification of non-essential hazardous heavy metals and metalloids in food chains/webs—Concepts and implications for wildlife and human health. <i>Human and Ecological Risk Assessment: An International Journal,</i> 25(6), 1353-1376. <br>
    2. Hutton, M. (1983). Sources of cadmium in the environment. <i>Ecotoxicology and environmental safety,</i> 7(1), 9-24. <br>
    3. Idrees, N., Tabassum, B., Abd_Allah, E. F., Hashem, A., Sarah, R., & Hashim, M. (2018). Groundwater contamination with cadmium concentrations in some West UP Regions, India. <i>Saudi Journal of Biological Sciences,</i> 25(7), 1365-1368. <br>
    4. Olsén, A., Wick, M. J., Mörgelin, M., & Björck, L. (1998). Curli, fibrous surface proteins of Escherichia coli, interact with major histocompatibility complex class I molecules. <i>Infection and immunity,</i> 66(3), 944-949. <br>
    5. Ravera, O. (1984). Cadmium in freshwater ecosystems. <i>Experientia,</i> 40(1), 1-14. <i>Infection and immunity,</i> 66(3), 944-949. <br>
    6. Brombacher, E., Baratto, A., Dorel, C., & Landini, P. (2006). Gene Expression Regulation by the Curli Activator CsgD Protein: Modulation of Cellulose Biosynthesis and Control of Negative Determinants for Microbial Adhesion. Journal of Bacteriology, 188(6), 2027-2037. <a class="text-dark" href="http://doi.org/10.1128/JB.188.6.2027-2037.2006">http://doi.org/10.1128/JB.188.6.2027-2037.2006</a>  <br>
    7. Evans, M. L., & Chapman, M. R. (2014). Curli biogenesis: order out of disorder. Biochimica et Biophysica Acta (BBA)-Molecular Cell Research, 1843(8), 1551-1558. <br>
    8. <a class="text-dark" href="https://www.uniprot.org/uniprotkb/P0AE22/entry">https://www.uniprot.org/uniprotkb/P0AE22/entry</a> <br>
    9. <a class="text-dark" href="http://2014.igem.org/Team:INSA-Lyon/Biology">http://2014.igem.org/Team:INSA-Lyon/Biology</a> <br>
    10. <a class="text-dark" href="http://2017.igem.org/Team:TAS_Taipei">http://2017.igem.org/Team:TAS_Taipei</a> <br>
    11. Martinez-Hackert, E., & Stock, A. M. (1997). The DNA-binding domain of OmpR: crystal structures of a winged helix transcription factor. Chemistry & Biology, 4(2), 162. doi:10.1016/s1074-5521(97)90269-6 <br>
    12. Olsén, A., Wick, M. J., Mörgelin, M., & Björck, L. (1998). Curli, fibrous surface proteins of Escherichia coli, interact with major histocompatibility complex class I molecules. Infection and immunity, 66(3), 944-949. <br>
    13. Reinke, A A, and J E Gestwicki. “Insight into amyloid structure using chemical probes.” Chemical biology & drug design., U.S. National Library of Medicine, June 2011, <a class="text-dark" href="www.ncbi.nlm.nih.gov/pubmed/21457473">www.ncbi.nlm.nih.gov/pubmed/21457473</a>. <br>
    14. Reisner, A., Haagensen, J. A., Schembri, M. A., Zechner, E. L., & Molin, S. (2003). Development and maturation of Escherichia coli K-12 biofilms. Molecular microbiology, 48(4), 933-946. <br>
    15. J. Xing, R. Hickey, Response in performance of the GAC-fluidized bed reactor process for BTX removal to perturbation in oxygen and nutrient supply, International Biodeterioration &  Biodegradation, Volume 33, Issue 1,1994, Pages 23-39, ISSN 0964-8305, <a href= "https://doi.org/10.1016/0964-8305(94)90053-1">https://doi.org/10.1016/0964-8305(94)90053-1</a>. <br>
    16. Magdum, Sandip & V., Kalyanraman. (2019). Evaluation of High Rate MBBR to Predict Optimal  Design Parameters for Higher Carbon and Subsequent Ammoniacal Nitrogen Removal. Current  science, 116. 2083-2088. 10.18520/cs/v116/i12/2083-2088. <br>
    17. Brackin, M.J., McKenzie, D.E., Hughes, B.M. et al. Laboratory-scale evaluation of fluidized bed  reactor technology for biotreatment of maleic anhydride process wastewater. Journal of  Industrial Microbiology 16, 216-223 (1996). <a href="https://doi.org/10.1007/BF01570024">https://doi.org/10.1007/BF01570024</a>. <br>
    18. Yu N, Zhang D, Lei Y, Wang J, Dong Y, Chen Y. Innovative Coating-Etching Method of Biocarrier  Fabrication for Treating Wastewater with a Low C/N Ratio. Polymers, 2022 Jul 25;14(15):3010.<br>
    19. J. Xing, R. Hickey, Response in performance of the GAC-fluidized bed reactor process for BTX removal to perturbation in oxygen and nutrient supply, <i>International Biodeterioration & Biodegradation,</i> Volume 33, Issue 1,1994, Pages 23-39, ISSN 0964-8305, https://doi.org/10.1016/0964-8305(94)90053-1. <br>
    20. Magdum, Sandip & V., Kalyanraman. (2019). Evaluation of High Rate MBBR to Predict Optimal Design Parameters for Higher Carbon and Subsequent Ammoniacal Nitrogen Removal. <i>Current science,</i> 116. 2083-2088. 10.18520/cs/v116/i12/2083-2088. <br>
    21. Brackin, M.J., McKenzie, D.E., Hughes, B.M. et al. Laboratory-scale evaluation of fluidized bed reactor technology for biotreatment of maleic anhydride process wastewater. <i>Journal of Industrial Microbiology 16,</i> 216-223 (1996). https://doi.org/10.1007/BF01570024. <br>
    22. Yu N, Zhang D, Lei Y, Wang J, Dong Y, Chen Y. Innovative Coating-Etching Method of Biocarrier Fabrication for Treating Wastewater with a Low C/N Ratio. <i>Polymers,</i> 2022 Jul 25;14(15):3010 <br>
    23. Chalfie, M., & Kain, S. R. (2005). Green fluorescent protein: Properties, applications and protocols. John Wiley & Sons. <br>
    24. Green fluorescent protein as a signal for protein-protein interactions - <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/pro.5560061107" target="_blank">https://onlinelibrary.wiley.com/doi/abs/10.1002/pro.5560061107</a>
    25. Park, S., & Raines, R. T. (2008). Green fluorescent protein as a signal for protein-protein interactions. Protein Science, 6(11), 2344-2349. <a href="https://doi.org/10.1002/pro.5560061107" target="_blank">https://doi.org/10.1002/pro.5560061107</a>
    26. Kukar, T., Eckenrode, S., Gu, Y., Lian, W., Megginson, M., She, J., & Wu, D. (2002). Protein microarrays to detect protein-protein interactions using red and green fluorescent proteins. Analytical Biochemistry, 306(1), 50-54. <a href="https://doi.org/10.1006/abio.2002.5614">https://doi.org/10.1006/abio.2002.5614</a>
    27. Lin, X., Li, Z., Li, Y., & Lu, Y. (2021). A robust escherichia coli cell-free expression toolbox driven by sigma factors. Biochemical Engineering Journal, 171, 108031. <a href="https://doi.org/10.1016/j.bej.2021.108031">https://doi.org/10.1016/j.bej.2021.108031</a>
    28. Blanco, L. P., Evans, M. L., Smith, D. R., Badtke, M. P., & Chapman, M. R. (2012). Diversity, biogenesis and function of microbial amyloids. Trends in Microbiology, 20(2), 66-73.<a href="https://doi.org/10.1016/j.tim.2011.11.005">https://doi.org/10.1016/j.tim.2011.11.005</a>
    29. Thaller, M. C., Schippa, S., Bonci, A., Cresti, S., & Rossolini, G. M. (2006). Identification of the gene (aphA) encoding the class B acid phosphatase/phosphotransferase of escherichia coli MG1655 and characterization of its product. FEMS Microbiology Letters, 146(2), 191-198. <a href="https://doi.org/10.1111/j.1574-6968.1997.tb10192.x">https://doi.org/10.1111/j.1574-6968.1997.tb10192.x</a>
    30. <a href="https://www.thermofisher.com/order/catalog/product/N301">https://www.thermofisher.com/order/catalog/product/N301</a> <br>
    31. Judith H. Merritt,Daniel E. Kadouri, George A. O’Toole. Growing and Analyzing Static Biofilms.2005. <a href="10.1002/9780471729259.mc01b01s00">10.1002/9780471729259.mc01b01s00</a> <br>
    32. <a href="http://parts.igem.org/Help:Assembly/3A_Assembly">http://parts.igem.org/Help:Assembly/3A_Assembly</a> <br>
    33. <a href="https://2021.igem.org/Team:IISER-Tirupati_India">https://2021.igem.org/Team:IISER-Tirupati_India</a> <br>
    34. <a href="https://2020.igem.org/Team:Calgary">https://2020.igem.org/Team:Calgary</a> <br>
    35. Ya-Wen Chang1, Alexandros A Fragkopoulos1, Samantha M Marquez2, Harold D Kim1, Thomas E Angelini3 and Alberto Fernández-Nieves1.Biofilm formation in geometries with different surface curvature and oxygen availability 6 March 2015. <br>
    36. <a href="https://www.addgene.org/protocols/gel-purification/">https://www.addgene.org/protocols/gel-purification/</a> <br>
    37. Pei Yun Lee,1 John Costumbrado,1 Chih-Yuan Hsu,1 and Yong Hoon Kim1.Agarose Gel Electrophoresis for the Separation of DNA Fragments.2012 .doi: 10.3791/3923 <br>
    38. C. Picioreanu, M.C.M van Loosdrecht, J.J. Heijnen, &rdquo;Two dimensional model of biofilm detachment caused by internal stress from liquid flow&rdquo;, <em>Biotechnology and Bioengineering</em>, vol. 72, no. 2, pp. 205-218, 2001. <a href="https://doi.org/10.1002/1097-0290(20000120)72:2%3C205::AID-BIT9%3E3.0.CO;2-L">https://doi.org/10.1002/1097-0290(20000120)72:2%3C205::AID-BIT9%3E3.0.CO;2-L</a> <br>
    39. Picioreanu, C., Van Loosdrecht, M. C., &amp; Heijnen, J. J. (1998). Mathematical modeling of biofilm structure with a hybrid differential‐discrete cellular automaton approach.&nbsp;<em>Biotechnology and bioengineering, 58</em>(1), 101-116. <a href="https://doi.org/10.1002/(SICI)1097-0290(19980405)58:1%3C101::AID-BIT11%3E3.0.CO;2-M">https://doi.org/10.1002/(SICI)1097-0290(19980405)58:1%3C101::AID-BIT11%3E3.0.CO;2-M</a> <br>
    40. Picioreanu, C., van Loosdrecht, M., &amp; Heijnen, J. (1999). <em>Multidimensional modeling of biofilm structure.</em> Delft University of Technology, Faculty of Applied Sciences. <br>
    41. Rittman, B. E. (1982). The effect of shear stress on biofilm loss rate. <em>Biotechnology and bioengineering, 24</em>(2), 501-506. <a href="https://doi.org/10.1002/bit.260240219">https://doi.org/10.1002/bit.260240219</a> <br>
    42. Proshkin, S., Rahmouni, A. R., Mironov, A., &amp; Nudler, E. (2010). Cooperation between translating ribosomes and RNA polymerase in transcription elongation. Science, 328(5977), 504-508. <a href="https://doi.org/10.1126/science.1184939">https://doi.org/10.1126/science.1184939</a> <br>
    43. Ledezma-Tejeida, D., Altamirano-Pacheco, L., Fajardo, V., &amp; Collado-Vides, J. (2019). Limits to a classic paradigm: most transcription factors in E. coli regulate genes involved in multiple biological processes. Nucleic acids research, 47(13), 6656-6667. <a href="https://doi.org/10.1101/479857">https://doi.org/10.1101/479857</a> <br>
    44. Kim, H., &amp; Gelenbe, E. (2011). Stochastic gene expression modeling with hill function for switch-like gene responses. IEEE/ACM Transactions on Computational Biology and Bioinformatics, 9(4), 973-979. (<a href="https://doi.org/10.1109/TCBB.2011.153">https://doi.org/10.1109/TCBB.2011.153</a>) <br>
    45. Likhoshvai, V., &amp; Ratushny, A. (2007). Generalized Hill function method for modeling molecular processes. Journal of bioinformatics and computational biology, 5(02b), 521-531. (<a href="https://doi.org/10.1142/S0219720007002837">https://doi.org/10.1142/S0219720007002837</a>) <br>
    46. Yang, J., Yan, R., Roy, A., Xu, D., Poisson, J., &amp; Zhang, Y. (2015). The I-TASSER Suite: protein structure and function prediction. <em>Nature methods,</em> 12(1), 7-8. (<a href="https://doi.org/10.1038/nmeth.3213">https://doi.org/10.1038/nmeth.3213</a>) <br>
    47. Roy, A., Kucukural, A., &amp; Zhang, Y. (2010). I-TASSER: a unified platform for automated protein structure and function prediction.&nbsp;<em>Nature protocols</em>, 5(4), 725-738. (<a href="https://doi.org/10.1038/nprot.2010.5">https://doi.org/10.1038/nprot.2010.5</a>) <br>
    48. Zhang, Y. (2008). I-TASSER server for protein 3D structure prediction.<em>&nbsp;BMC bioinformatics</em>, 9(1), 1-8. (<a href="https://doi.org/10.1186/1471-2105-9-40">https://doi.org/10.1186/1471-2105-9-40</a>)
    49. Jumper, J., Evans, R., Pritzel, A., Green, T., Figurnov, M., Ronneberger, O., ... &amp; Hassabis, D. (2021). Highly accurate protein structure prediction with AlphaFold. Nature, 596(7873), 583-589. (<a href="https://doi.org/10.1038/s41586-021-03819-2">https://doi.org/10.1038/s41586-021-03819-2</a>) <br>
    50. Mirdita, M., Sch&uuml;tze, K., Moriwaki, Y., Heo, L., Ovchinnikov, S., &amp; Steinegger, M. (2022). ColabFold: making protein folding accessible to all. Nature Methods, 1-4. (<a href="https://doi.org/10.1038/s41592-022-01488-1">https://doi.org/10.1038/s41592-022-01488-1</a>) <br>
    51. Jumper, J., Evans, R., Pritzel, A., Green, T., Figurnov, M., Ronneberger, O., ... &amp; Hassabis, D. (2021). Highly accurate protein structure prediction with AlphaFold. Nature, 596(7873), 583-589. (<a href="https://doi.org/10.1038/s41586-021-03819-2">https://doi.org/10.1038/s41586-021-03819-2</a>) <br>
    52. Mirdita, M., Sch&uuml;tze, K., Moriwaki, Y., Heo, L., Ovchinnikov, S., &amp; Steinegger, M. (2022). ColabFold: making protein folding accessible to all. Nature Methods, 1-4.(<a href="https://doi.org/10.1038/s41592-022-01488-1">https://doi.org/10.1038/s41592-022-01488-1</a>) <br>
    53. Schrödinger, L., & DeLano, W. (2020). PyMOL. Retrieved from <a href="http://www.pymol.org/pymol">PyMOL</a> <br>
    54. Khaled Obaideen, Nabila Shehata, Enas Taha Sayed, Mohammad Ali Abdelkareem, Mohamed S. Mahmoud, A.G. Olabi, The role of wastewater treatment in achieving sustainable development goals (SDGs) and sustainability guideline, Energy Nexus, Volume 7, 2022,100112, ISSN 2772-4271, <a href="https://doi.org/10.1016/j.nexus.2022.100112" target="blank">https://doi.org/10.1016/j.nexus.2022.100112</a> <br>
    55. Tortajada, C. Contributions of recycled wastewater to clean water and sanitation Sustainable Development Goals. npj Clean Water 3, 22 (2020). <a href="https://doi.org/10.1038/s41545-020-0069-3" target="_blank">https://doi.org/10.1038/s41545-020-0069-3</a> <br>
    56. Alex J. Zautra, Anne Arewasikporn & Mary C. Davis (2010) Resilience: Promoting Well-Being Through Recovery, Sustainability, and Growth, Research in Human Development, 7:3, 221-238, DOI: 10.1080/15427609.2010.504431. <br>
    57. Janice Denoncourt (2020) Companies and UN 2030 Sustainable Development Goal 9 Industry, Innovation and Infrastructure, Journal of Corporate Law Studies, 20:1, 199-235, DOI: 10.1080/14735970.2019.1652027 <br>
    58. Arora, N.K., Mishra, I. United Nations Sustainable Development Goals 2030 and environmental sustainability: race against time. <i>Environmental Sustainability</i> 2, 339-342 (2019). <a href="https://doi.org/10.1007/s42398-019-00092-y" target="_blank">https://doi.org/10.1007/s42398-019-00092-y</a> <br>
    59. Mcquarrie, James & Boltz, Joshua. (2011). Moving Bed Biofilm Reactor Technology: Process Applications, Design, and Performance. Water environment research : a research publication of the Water Environment Federation. 83. 560-75. 10.2175/106143010X12851009156286.
  </p>
</div>

{% endblock %}