{% extends "layout.html" %}
  
{% block title %}Contribution{% endblock %}
{% block bnrurl %}https://static.igem.wiki/teams/4509/wiki/banners/contribution.jpeg{% endblock %}


{% block page_content %}


<h4>Overview</h4>
<p>
  Making a valuable contribution to future iGEM Teams has been our main impetus since the beginning  of our project. We benefited greatly from what our predecessors transcribed in their wiki, and we  have decided to do our best to be as useful to future generations of iGEM students as possible. 
</p>

<h4>Protein-Coding Sequence Modification</h4>

<p>
  The class B acid phosphatase enzyme coding sequence obtained from NCBI for the project was regrettably incompatible with the BioBrick RFC[10].
</p>
<center><img src="https://static.igem.wiki/teams/4509/wiki/before-opt.png" style="margin-top: 1em; width:65%; height:500px;"/></center>
<div style="margin-top: 2em; margin-bottom: 1em; text-align: center;">Figure 1: Acid phosphatase(aphA) coding sequence</div>
<div style="margin-top: 1em; margin-bottom: 1em; text-align: center;"> When the sequence from NCBI is uploaded to the iGEM registry, it is evident that the sequence is not compatible with other Biobrick or iGEM standards except RFC[1000]</div>

<p>
  Therefore, we decided to troubleshoot this by removing the illegal restriction sites by inducing silent mutations by use of IDT's codon optimizing tool. As a result, the protein that is expressed would remain the same while enabling the sequence to comply with the iGEM assembly standard requirement.
</p>

<center><img src="https://static.igem.wiki/teams/4509/wiki/idt-tool.jpeg" style="margin-top: 1em; width:65%; height:500px;"/></center>

<div style="margin-top: 2em; margin-bottom: 1em; text-align: center;">Figure 2: IDT Codon Optimizing Tool</div>
<div style="margin-top: 1em; margin-bottom: 1em; text-align: center;"> The figure indicates the illegal restriction sites present in the protein coding sequence</div>

<p>We removed all the illegal restriction sites for the protein-coding sequences of acid phosphatase to be compatible with the BioBrick RFC[10]. We believe that this contribution will make assembly more convenient and universal for future iGEM teams who plan on assembling or synthesizing acid phosphatase in a prokaryotic system.</p>

<center><img src="https://static.igem.wiki/teams/4509/wiki/after-opt.png" style="margin-top: 1em; width:65%; height:500px;"/></center>
<div style="margin-top: 2em; margin-bottom: 1em; text-align: center;">Figure 3: Optimized Acid Phosphatase(aphA) Sequence</div>
<div style="margin-top: 1em; margin-bottom: 1em; text-align: center;">The codon-optimized sequence is now compatible with the Biobrick or iGEM standards. </div>
<p>
  For more information, please visit our <a href="{{ url_for('pages', page='parts') }}">parts </a> page!
</p>

<h4>Part Improvement</h4>

<p>
  Our project Curlim requires increased production of curli fiber for biofilm formation, which guided us to the composite part 
  <a href="http://parts.igem.org/Part:BBa_K2229300" target="_blank">BBa_K2229300</a> developed by iGEM17_TAS_Taipei. REC-CHENNAI's review studies routed us to discover the burden imparted on the 
  cell by the existing promoter. The promoter <a href="http://parts.igem.org/Part:BBa_J23100" target="_blank">BBa_J23100</a> offers a strength of 1 au and a burden value in the range of 20.0% ± 9.9%.
   Studies showed that the burden value of the RBS(B0034) - strong promoter(J23100) amalgamated to be 27.5 ± 8.6%. Thus it is vital to note that biobricks with a burden value >20% - 30% will result in reduced protein synthesis,may be susceptible to becoming less functional or nonfunctional units, and also cause cell death. Considering the above factors 
   we mindfully chose <a href="http://parts.igem.org/Part:BBa_J23118" target="_blank"> BBa_J23118</a>, a strong promoter from the Anderson Series. This promoter expresses the strength of 0.56 au and produces a burden value between the range of -0.1%±3.8% on the cell. On a final note, we replaced the promoter <a href="http://parts.igem.org/Part:BBa_J23100" target="_blank"> BBa_J23100 </a> with <a href="http://parts.igem.org/Part:BBa_J23100" target="_blank"> BBa_J23118 </a>, thus creating an improved composite part <a href="http://parts.igem.org/Part:BBa_J23100" target="_blank"> BBa_K4509469</a>.
</p>

<h4>Adding New Parts</h4>

<p>
  While working on project CURLIM, our team added the upcoming new basic and composite parts to the iGEM registry. The biofilm formation and dephosphorylation processes will be facilitated by the following components.
</p>

<h6>
  <a href="http://parts.igem.org/Part:BBa_K4509669" target="_blank">BBa_K4509669</a>
</h6>
<p>
  It employs a metal sensing promoter to help produce acid phosphatase when cadmium is present.
</p>

<h6>
  <a href="http://parts.igem.org/Part:BBa_K4509769" target="_blank">BBa_K4509769</a>
</h6>
<p>
  It promotes the synthesis of acid phosphatase as cell surface protein.
</p>

<h6> 
  <a href="http://parts.igem.org/Part:BBa_K4509469" target="_blank">BBa_K4509469</a>
</h6>

<p>
  It promotes the formation of curli fiber by overexpressing the proteins csgD and OmpR234.
</p>

<h4>Literature Reference</h4>

<p>
  The production of more curli fibers for biofilm formation, the synthesis of acid phosphatase enzyme, and the tagging of a reporter gene to identify DNA sequences or monitor gene expression are some of the significant activities carried out by the CURLIM. 
  During this phase, we came to understand the necessity and significance of detailed literature evaluations pertaining to the functionality, efficacy, applicability, etc., of the components of the project. 
  Literature evidence supported us and helped us solve problems during multiple investigations. For the benefit of upcoming iGEM teams and researchers, we have included a few references to the parts <a href="http://parts.igem.org/Part:BBa_K259006" target="_blank"> BBa_K259006</a> (GFP-Terminator), <a href="http://parts.igem.org/Part:BBa_k1404006" target="_blank">BBa_k1404006</a> (p70-CsgA, curli generator), and <a href="http://parts.igem.org/Part:BBa_K4509369" target="_blank">BBa_K4509369</a> (Acid phosphatase (aphA)).
</p>


<h4>Team And Working Structure</h4>

<p>
  Our team has 15 members, which is a large number that necessitates a structured and organized team framework for effective operation. 
  So, in this section, we present our team's learning, practices, and establishments in order to assist future iGEMers in forming a successful and efficient team. 
  The flow below depicts a detailed and structured team framework.
</p>

<h5>Step 1 - Planning</h5>

<p>
  At first, the team chose suitable leaders who were in charge of overseeing the team operations, competition requirements, and deadlines.
</p>

<p>
  Second, the team brainstormed all of the project requirements and categorized each team member's strengths in order to assign the right person to the right task. 
  They were divided into subgroups based on their individual strengths and project requirements. 
  Every member had to understand the entire project's working process, so the teams were shuffled regularly to ensure equal contribution from all.
</p>

<p>
  Every subgroup had a leader who assigned tasks to members and mediated communication between subgroup members, the team leader, and other subgroups.
</p>

<h5>Step 2 - Communication</h5>

<p>
  Communication is essential for project success, so team members should communicate on a regular basis to stay informed about the project's progress.
</p>

<ol>
  <li>Communication between team members:</li>
  <ul>
    <li>
      <p>
        Our team established protocols from the start to ensure productive team meetings. 
        An hour before the scheduled meeting, the subgroup would circulate a detailed agenda in the form of digital documents to every other subgroup outlining their workflow of the project, future requirements, and other points to be discussed. 
        This practice saved time and kept discussions on track. During the discussion, "Minutes of the Meeting - MoM" were jotted down, which included the date, time, attendees, and pointers discussed, which significantly reduced the time we spent on the wiki content.
      </p>
    </li>
  </ul>

  <li>Communication between mentors:</li>
  <ul>
    <li>
      <p>
        Every Friday of the week, we would host a meeting with our mentors where the details of the team meetings would be presented in the form of digital documents and they would suggest improvisations to fix our troubleshoots and help us progress in our work.
      </p>
    </li>
  </ul>
</ol>

<h5> Step 3 - Organize and Implement </h5>

<p>
  The team would record the developments and tasks (wet lab, dry lab, outreach, etc.) scheduled for the following week based on the meetings in a "Team Register". 
  This register also contains information about tasks assigned to a specific member, task protocols, task requirements, and so on. 
  Upon completion of the task, the member would record the data in a digital document and distribute it to the rest of the team members and to the mentors.
</p>
<p>
  We are confident that strict adherence to these three standards resulted in excellent team coordination and effective results. 
  As a result, we present our successful team strategy for the benefit of future iGEM teams.
</p>

<div style="text-align: center;"> Best wishes!</div>

<h4 style="margin-top: 2em;"> References</h4>

<ol style="word-wrap: break-word">
  <li>
    <p style="word-wrap: break-word">
      Chalfie, M., & Kain, S. R. (2005). Green fluorescent protein: Properties, applications and protocols. John Wiley & Sons.
    </p>
  </li>
  <li>
    <p style="word-wrap: break-word">
      Green fluorescent protein as a signal for protein-protein interactions - 
      <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/pro.5560061107" target="_blank">
        https://onlinelibrary.wiley.com/doi/abs/10.1002/pro.5560061107 
      </a>
    </p>
  </li>
  <li>
    <p style="word-wrap: break-word">
      Park, S., & Raines, R. T. (2008). Green fluorescent protein as a signal for protein-protein interactions. Protein Science, 6(11), 2344-2349. 
      <a href="https://doi.org/10.1002/pro.5560061107" target="_blank">
        https://doi.org/10.1002/pro.5560061107
      </a>
    </p>
  </li>
  <li>
    <p style="word-wrap: break-word">
      Kukar, T., Eckenrode, S., Gu, Y., Lian, W., Megginson, M., She, J., & Wu, D. (2002). Protein microarrays to detect protein-protein interactions using red and green fluorescent proteins. Analytical Biochemistry, 306(1), 50-54. 
    <a href="https://doi.org/10.1006/abio.2002.5614">
      https://doi.org/10.1006/abio.2002.5614
    </a>
    </p>
  </li>
  <li>
    <p style="word-wrap: break-word">
      Lin, X., Li, Z., Li, Y., & Lu, Y. (2021). A robust escherichia coli cell-free expression toolbox driven by sigma factors. Biochemical Engineering Journal, 171, 108031. 
      <a href="https://doi.org/10.1016/j.bej.2021.108031">
        https://doi.org/10.1016/j.bej.2021.108031
      </a>
    </p>
  </li>
  <li>
    <p style="word-wrap: break-word">
      Blanco, L. P., Evans, M. L., Smith, D. R., Badtke, M. P., & Chapman, M. R. (2012). Diversity, biogenesis and function of microbial amyloids. Trends in Microbiology, 20(2), 66-73.
      <a href="https://doi.org/10.1016/j.tim.2011.11.005">
        https://doi.org/10.1016/j.tim.2011.11.005
      </a>
    </p>
  </li>
  <li>
    <p style="word-wrap: break-word">
      Thaller, M. C., Schippa, S., Bonci, A., Cresti, S., & Rossolini, G. M. (2006). Identification of the gene (aphA) encoding the class B acid phosphatase/phosphotransferase of escherichia coli MG1655 and characterization of its product. FEMS Microbiology Letters, 146(2), 191-198.
      <a href="https://doi.org/10.1111/j.1574-6968.1997.tb10192.x">
        https://doi.org/10.1111/j.1574-6968.1997.tb10192.x
      </a>
    </p>
  </li>
</ol>

{% endblock %}
